metrics/collectd/collectd.conf

```diff
@@ -17,7 +17,7 @@ Hostname localhost
     ValuesPercentage true
 </Plugin>
 <Plugin "csv">
-    DataDir "/opt/collectd"
+    DataDir "/mnt/opt/collectd/run"
```

Comment:
I don't see how `/mnt` is different from `/opt` - what am I missing? I don't see it put up as a volume in the pod for instance - probably some defacto setup or default I don't know about?

Reply:
The collectd container has `host:/opt` mounted to `container:/mnt/opt` ([collectd.yaml](https://github.com/clearlinux/cloud-native-setup/blob/a0ca2a2017cb36624d3e34b846161a243d73a56c/metrics/collectd/collectd.yaml#L44)). `container:/opt`, where collectd has written the results so far, is located on overlay.